doc/whatsnew.rst

```diff
@@ -4,6 +4,17 @@ What's new
 Next release
 ============
 
+  Water/Nexus
```

Comment:
@Wegatriespython @glatterf42 while reviewing #388 I noticed that this heading and the bullet list below have a leading indent of two spaces. In the built HTML docs, the heading does not appear and the list items don't match the rest of the list.

I'll correct the change in a commit on that branch, to align with the style used for previous releases.

In the future, please closely eyeball docs additions to make sure they render as you intend. The rest of the same file can be an example for formatting.